pkg/sdk/go/sdk.go

```diff
@@ -256,6 +266,15 @@ type SDK interface {
 
 	// RevokeCert revokes certificate with certID for thing with thingID
 	RevokeCert(thingID, certID, token string) error
+
+	// Issue issues a new Key, returning its token value alongside.
```

Comment:
Do we need to capitalize `Key` in this comment? Maybe simple `key` would be more appropriate?